hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3131,3 +3156,142 @@ func reconcileMachineApproverDeployment(deployment *appsv1.Deployment, hc *hyper
 	hyperutil.SetDefaultPriorityClass(deployment)
 	return nil
 }
+
+const oidcDocumentsFinalizer = "hypershift.io/aws-oidc-discovery"
```

Comment:
Any reason we can't use existing `finalizer = "hypershift.openshift.io/finalizer"?

Reply:
nvm ignore me.